examples/vue/src/stories/index.js

```diff
@@ -96,4 +98,15 @@ storiesOf('Addon Notes', module)
   ))
   .add('with some bold text', withNotes({ notes: 'My notes on some bold text' })(() => ({
     render: h => h('div', [h('strong', ['A very long text to display'])])
-  })));
\ No newline at end of file
+  })));
+
+
+  storiesOf('Addon Knobs', module)
+    .addDecorator(withKnobs)
```

Comment:
Yes it does make sense to have the same api. But possibly the current api doesn't provide the right abstraction, we're still investigating into this.